stubs/core.stub

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * @param mixed $var
+ * @param bool $return
+ * @return ($return is true ? string : void)
```

Comment:
when running tests on phpstan-src@1.6.x it seems to make a different whether the `@return` expression here is sorrounded by braces.. I think this is not expected

Reply:
after reading the 1.6.x release blog post, I conclude the paranthesis are necessary.